proto/ibc/core/channel/v1/upgrade.proto

```diff
@@ -23,5 +23,5 @@ message ErrorReceipt {
   // the channel upgrade sequence
   uint64 sequence = 1;
   // the error message detailing the cause of failure
-  string error = 2;
+  string error_message = 2;
```

Comment:
Nit: since the type name already indicates this is an error, maybe the field can be just `message`?

```suggestion
  string message = 2;
```

Reply:
yeah I'm happy enough to make this change